impeller/renderer/backend/vulkan/blit_pass_vk.cc

```diff
@@ -35,7 +36,11 @@ bool BlitPassVK::EncodeCommands(
     return false;
   }
 
-  auto encoder = encoder_.lock();
+  auto command_buffer = command_buffer_.lock();
+  if (!command_buffer) {
+    return false;
+  }
+  auto encoder = command_buffer->GetEncoder();
```

Comment:
I've updated RenderPass/BlitPass/And ComputePass to be non-const so that the first time command_buffer->GetEncoder() is called we construct the VkCommandEncoder.